doc/timeout_overrides.md

```diff
@@ -0,0 +1,69 @@
+# Timeout overrides
+
+Jibri is using several timeouts, which allow it to control and monitor the recording. It is possible to adjust the timeout values and tailor its operations to more specific requirements.
+
+The configuration options are set in the configuration file. Both versions of the configuration file (HOCON and JSON) are supported.
+
+### No Media Timeout
+If all clients have their audio and video muted and if Jibri does not detect any data stream (audio or video) comming in, it will stop recording after `NO_MEDIA_TIMEOUT` expires.
```

Comment:
We can get rid of this doc and put this documentation above the fields in `reference.conf`